File: C/unit_3/Fg.c
#include<stdio.h>
int main()
{
   int a1,a2,a3;
   printf("Enter the 1st Angle of the Traingle !!!!");
   scanf("%d",&a1);
   printf("Enter the 2nd Angle of the Traingle !!!");
   scanf("%d",&a2);
   printf("Enter the 3rd Angle of the Traingle !!!");
   scanf("%d",&a3);
   if(a1+a2+a3==180)
    printf("It is a Valid Traingle!!!");
   else 
    printf("It is an Invalid Traingle !!!");
   return 0;
}

File: Java/unit_3/Fg.java
package Java.unit_3;

import java.util.Scanner;

public class Fg {
    public static void main(String[] args) {
        double a1, a2, a3;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the 1st angle of the Traingle : \n");
        a1 = sc.nextDouble();
        System.out.println("Enter the 2nd angle of the Traingle : \n");
        a2 = sc.nextDouble();
        System.out.println("Enter the 3rd angle of the Traingle : \n");
        a3 = sc.nextDouble();
        if (a1 + a2 + a3 == 180)
            System.out.println("It is a Valid Traingle !!! ");
        else
            System.out.println("It is an Invalid Traingle !!! ");
        sc.close();
    }
}
